<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbSettings</class>
 <widget class="QDialog" name="DbSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Database</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Hostname</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="le_username">
         <property name="text">
          <string>faqadmin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_password">
         <property name="text">
          <string>222&amp;autoDesk</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_database">
         <property name="text">
          <string>faqtooldb</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_host">
         <property name="text">
          <string>postgresql-10318-0.cloudclusters.net</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_port">
         <property name="text">
          <string>10318</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btn_set_app">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
